Sources/SectionScrubber.swift

```diff
@@ -253,6 +255,12 @@ public class SectionScrubber: UIView {
         }
         self.sectionLabel.hide()
     }
+
+    override public func layoutSubviews() {
+        self.containingViewFrame = CGRectMake(0, 64, self.collectionView.bounds.width, self.collectionView.bounds.height - 64 - self.viewHeight)
```

Comment:
I don't really like that the offset is hardcoded here. To me it seems that this should be controllable and visible for the client. Maybe we can make this default to the collectionViewBounds but overidable by the client 🤔